backend-tests/tests/test_rbac_v2.py

```diff
@@ -471,19 +588,32 @@ def test_deploy_to_group(self, clean_mongo, test_case):
                         "permission_sets_with_scope": [
                             {
                                 "name": "DeployToDevices",
-                                "scope": {"type": "DeviceGroups", "value": ["test"],},
-                            },
-                            {"name": "ManageTokens",},
+                                "scope": {"type": "DeviceGroups", "value": ["test"]},
+                            }
                         ],
-                    },
+                    }
                 ],
                 "device_groups": {"test": 1, "production": 1},
-                "deploy_group": "test",
-                "set_configuration_status_code": 204,
-                "deploy_configuration_status_code": 200,
+                "deploy_group": "production",
+                "set_configuration_status_code": 403,
+                "deploy_configuration_status_code": 403,
             },
             {
-                "name": "Test RBAC configuration deployment forbidden",
+                "name": "Test RBAC configuration deployment forbidden for RBAC_ROLE_OBSERVER",
+                "use_personal_access_token": False,
+                "user": {
+                    "name": "test1-UUID@example.com",
+                    "pwd": "password",
+                    "roles": ["RBAC_ROLE_OBSERVER"],
+                },
+                "roles": [],
+                "device_groups": {"test": 1, "production": 1},
+                "deploy_group": "production",
+                "set_configuration_status_code": 403,
+                "deploy_configuration_status_code": 403,
+            },
+            {
+                "name": "Test RBAC configuration deployment forbidden for ConfigureDevices permission set",
```

Comment:
would be good to also check if the configure permission allows to set the configuration

Reply:
Thanks, done!